sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CommonCoderTest.java

```diff
@@ -208,6 +214,22 @@ private static Object convertValue(Object value, CommonCoder coderSpec, Coder co
               convertValue(element, coderSpec.getComponents().get(0), elementCoder));
         }
         return convertedElements;
+      case "urn:beam:coders:global_window:0.1":
+        return GlobalWindow.INSTANCE;
+      case "urn:beam:coders:windowed_value:0.1":
+        Map<String, Object> kvMap = (Map<String, Object>) value;
+        Coder valueCoder = ((WindowedValue.FullWindowedValueCoder) coder).getValueCoder();
+        Coder windowCoder = ((WindowedValue.FullWindowedValueCoder) coder).getWindowCoder();
+        Object windowValue = convertValue(
+            kvMap.get("value"), coderSpec.getComponents().get(0), valueCoder);
+        Instant timestamp = new Instant(((Number) kvMap.get("timestamp")).longValue());
+        List<BoundedWindow> windows = new LinkedList<>();
+        for (Object window: ((List<Object>) kvMap.get("windows"))) {
+          windows.add((BoundedWindow) convertValue(window, coderSpec.getComponents().get(1),
+              windowCoder));
+        }
+        // Note: Until Python SDK supports PaneInfo, we default to PaneInfo.NO_FIRING.
+        return WindowedValue.of(windowValue, timestamp, windows, PaneInfo.NO_FIRING);
```

Comment:
Suggest that:

1. document the Python issue with a JIRA.
1. Restrict the test values to those with PaneInfo.NO_FIRING, and add a TODO in `standard_coders.yaml` referencing the JIRA.
1. return the correct value in Java.

Seems cleaner -- fewer places to clean up once Python has PaneInfo.